doc/ref_model/chapters/appendix-a.md

```diff
@@ -67,6 +70,35 @@ Without arguing for or against SR-IOV, CNTT provides the following anecdotes whi
 **CPU Instruction Sets and Extensions:**
 <content needed>
 
+**Raw Disk Device Mapping**
+<content needed>
+
+Overview of technology:
+
+It is possible to map SAN LUNs or local disks to a virtual unit (VM or container), giving direct hardware access to the
+specified device.  As this is a form of PCI passthrough, it should be considered as a violation of the abstration
+principle.
+
+See: vfio-pci for qemu.
```

Comment:
@markbeierl Hi Mark, Thanks for the input. I can easily see a use for this, and one thing to know about adding any features is even though the use may seem obvious, please be sure to include a _**rationale**_ for including the feature. There are lots of features we could add, but we try only to add the ones with a defendable rationale in the CNTT context. 

WRT PCI-PT, if you read the section carefully, there is no prohibition on PCI-PT, per se. The prohibition is on hardware-specific device drivers in the workload. However, you present an interesting case b/c it may or may not require special h/w drivers, depending on the ABI presented (raw block devices can be primitive or complex), and hence may or may not violate the rule. However, the local disk case may be transgressing a different concern, that we haven't captured. Specifically, if you anchor to a local piece of h/w (i.e. storage), even if no special drivers are required, you're still impeding portability. We don't currently have a prohibition on duct taping your workload to a physical server, even if there are no other concerns. The counter argument is, in some cases using local storage may be required for latency reasons, and we _have_ discussed that purely in the context of storage options, irrespective of the cloud-native discussion. If we choose to prohibit such associations, we can potentially mitigate by favoring other features you mention, such as RDMA or NVMeoF. @rabi-abdel Thoughts? @ASawwaf Comments?

Reply:
There are SW Def Storage use cases like CDN VOD with content on local drives (sizable content, can but doesn't have to  be latency sensitive) when VM cannot simply be instantiated on another node.